src/components/DemoBanner/DemoBanner.tsx

```diff
@@ -0,0 +1,135 @@
+import ExternalLink from "@saleor/components/ExternalLink";
+import { makeStyles } from "@saleor/macaw-ui";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { emphasisedTextBlue, greyDark, textColor } from "./constants";
+export const mediumScreen = 720;
+
+const useStyles = makeStyles(
+  theme => ({
+    wrapper: {
+      width: "100%",
+      backgroundColor: "inherit",
+      padding: `0 13px`,
+      fontSize: theme.spacing(1.5)
+    },
+    borderedWrapper: {
+      display: "flex",
+      flexDirection: "row",
+      justifyContent: "space-between",
+      alignItems: "center",
+      maxHeight: theme.spacing(7.5),
+      flex: "0 0 auto",
+      padding: theme.spacing(3, 0),
+      backgroundImage: `linear-gradient(to right,  ${greyDark} 13%, rgba(255, 255, 255, 0) 0%)`,
+      backgroundPosition: "bottom",
+      backgroundSize: theme.spacing(1.5, 0.125),
+      backgroundRepeat: "repeat-x"
+    },
+    logoWrapper: {
+      lineHeight: 0
+    },
+    linkList: {
+      display: "flex",
+      alignItems: "center"
+    },
+    link: {
+      display: "flex",
+      padding: theme.spacing(2),
+      color: textColor
+    },
+    textEmphasis: {
+      color: emphasisedTextBlue,
+      textTransform: "uppercase",
+      fontWeight: 600,
+      paddingLeft: "5px"
+    },
+    divider: {
+      borderRight: `1px solid  ${greyDark}`,
+      height: theme.spacing(2)
+    },
+    mediaMedium: {
+      display: "flex",
+      [theme.breakpoints.up(mediumScreen)]: {
```

Comment:
Not the same breakpoints. Material UI md is 960px. We need 720px.

Reply:
We need precisely `720px`, because it's where layout breaks, it's specified somewhere in ticket/design, or this component was just written in different project with different breakpoints?